fastlane/docs
# Group auto-generated files in one directory

### Motivation and context
`actions.md`, actions documentation and `available-plugins.md` are moved into `/generated` directory as proposed in https://github.com/fastlane/docs/pull/1075#issuecomment-895661648

**This PR should be considered together with:** https://github.com/fastlane/fastlane/pull/19309
 

### Description

Urls in actions documentation will change from:
- actions to generated/actions/<ACTION>, e.g.:
https://docs.fastlane.tools/actions/deliver to https://docs.fastlane.tools/generated/actions/deliver
- https://docs.fastlane.tools/actions/ to https://docs.fastlane.tools/generated/actions/
- https://docs.fastlane.tools/plugins/available-plugins to https://docs.fastlane.tools/generated/available-plugins/

Changes:
- Moved `actions.md` to `generated` directory
- Moved actions documentation to `generated/actions` directory
- Moved `plugins/available-plugins.md` to `generated` directory
- Updated `mkdocs.yml` - also actions list that will be regenerated by fastlane's `markdown_docs_generator`
- Updated documentation with  `bundle exec fastlane update_docs debug:true skip_plugin_scores:true` (https://github.com/fastlane/fastlane/blob/9b6d47e74da8cf05a8993ec8b4c2c66a466ce358/fastlane/Fastfile#L400)
- Updated theme (breadcrumbs)
- Removed `edit on github` button from available plugins docs
- Updated `available-redirects`
- Updated deployment script

### Testing Steps

Install mkdocs locally, build site, check html pages. Check links on pages.

I did initial testing of links. I couldn't test deployment.
